### DISCUSSION TITLE
Error in data fromats

### DISCUSSION THREAD

**Discussion by WolfDragon457** [PrismarineJS/bedrock-protocol#365] [Q&A]
Help, an error occurs:
 
`TypeError: SizeOf error for undefined : Cannot mix BigInt and other types, use explicit conversions`
 
When executing the Move Player package
 
```
player.upstream.queue('move_player', {
    runtime_id: ptick,
    position: {x: 9.518291473388672, y: 67.62001037597656, z: -0.6351199150085449},
    pitch: 9.698684692382812,
    yaw: -14.1260986328125,
    head_yaw: -14.1260986328125,
    mode: 'normal',
    on_ground: true,
    ridden_runtime_id: 0,
    teleport: { cause: 'unknown', source_entity_type: 0 },
    tick: 0
});
```
 
### Tell me, and then I've been suffering with this for half a day:(

**Comment by extremeheat** [comment]
This error is because you are supplying a 64bit number as a JS Number as opposed to a BigInt in one of the fields.

**Comment by WolfDragon457** [comment]
> This error is because you are supplying a 64bit number as a JS Number as opposed to a BigInt in one of the fields.

Solved, thank you very much
